src/libraries/Directory.Build.targets

```diff
@@ -231,6 +232,9 @@
 
   <PropertyGroup>
     <ExcludeFromPackage Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', '$(NetCoreAppCurrent)')) and '$(ExcludeCurrentNetCoreAppFromPackage)' == 'true'">true</ExcludeFromPackage>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>.dll;.exe;.winmd;.json;.pri;.xml;</AllowedOutputExtensionsInPackageBuildOutputFolder>
```

Comment:
Why is this necessary? I see that property already being defined with the same value in NuGet.targets: https://github.com/NuGet/NuGet.Client/blob/8eefb1e7d99c2992497b929c51e54b3c738b3e3b/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L48

Reply:
This is necessary to avoid treating pdbs as build output and stop them from including in the packages.

Reply:
Wouldn't we need to use this property to produce a symbol package and that way the pdbs are not in the regular package? https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#includesymbols

Reply:
no, IncludeSymbols property enables building another package with symbols in it.
arcade changes the value of this property ```AllowedOutputExtensionsInPackageBuildOutputFolder```  from the default

Reply:
Right, but that is a fallback fo `IncludeSymbols` not being set to true: https://github.com/dotnet/arcade/blob/ca7af8d541882be495d710b0aaf28276110aa8fc/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.targets#L59